unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -702,16 +704,15 @@ loop = do
                 _ -> do
                   (ppe, outputDiff) <- diffHelper before after
                   respondNumbered $ ShowDiffNamespace beforep afterp ppe outputDiff
-            CreatePullRequestI baseRepo headRepo -> unlessGitError do
-              (cleanupBase, baseBranch) <- viewRemoteBranch baseRepo
-              (cleanupHead, headBranch) <- viewRemoteBranch headRepo
-              lift do
-                merged <- eval $ Merge Branch.RegularMerge baseBranch headBranch
-                (ppe, diff) <- diffHelper (Branch.head baseBranch) (Branch.head merged)
-                respondNumbered $ ShowDiffAfterCreatePR baseRepo headRepo ppe diff
-                eval . Eval $ do
-                  cleanupBase
-                  cleanupHead
+            CreatePullRequestI baseRepo headRepo -> do
```

Comment:
Had to make either-handling more explicit because `viewRemoteBranch`'s are called within each other, and it needs to be `MonadUnliftIO`, so it can't use ExceptT.